views/export/index.tt

```diff
@@ -0,0 +1,106 @@
+[%# export/index.tt
+
+Journal export functionality
+
+Authors:
+    Mark Smith <mark@dreamwidth.org>
+
+Copyright (c) 2015 by Dreamwidth Studios, LLC.
+
+This program is free software; you may redistribute it and/or modify it under
+the same terms as Perl itself.  For a copy of the license, please reference
+'perldoc perlartistic' or 'perldoc perlgpl'.
+
+%][%- sections.title = title_ml | ml -%]
+
+[%- CALL dw.ml_scope( '/export.bml' ) -%]
+[%- CALL dw.active_resource_group( "foundation" ) -%]
+[% sections.title = '.title' | ml %]
+
+[%- authas_form -%]
+
+<p>[% '.description' | ml %]</p>
+
+<form action='export_do?authas=[% u.username %]' method='post'>
+
+<fieldset>
+
+<div class="row">
+    <div class="large-2 columns"></div>
+    <div class="large-2 columns"><label>[% '.label.what' | ml %]</label></div>
```

Comment:
To align the label better, you can use `label.inline` from http://foundation.zurb.com/docs/components/forms.html

But -- unless there's a very strong reason to do it this way, I would strongly suggest keeping it consistent with the rest of the site by having the label stack vertically. Otherwise, we're right back where we started with each form on each page having a different look and feel.

I do see what you mean about not wanting the form to be too large though, but for that I would suggest two key fixes:

* remove the "export what" dropdown. We only have one option! We don't need to offer a choice.
* we could make the "Fields" checkboxes go into two columns, too